packages/now-cli/src/util/error.ts

```diff
@@ -25,19 +42,16 @@ export async function responseError(
     message = bodyError.message;
   }
 
-  if (message == null) {
+  if (!message) {
```

Comment:
Can this whole if statement be simplified?

```js
const message = bodyError.message || fallbackMessage || 'Response Error';
```

Or do we want an error message with an empty string in some cases?

Reply:
No, I don't think that's desirable